awscli/customizations/wizard/wizards/configure/_main.yml

```diff
@@ -0,0 +1,113 @@
+version: "0.1"
+title: Configure the AWS CLI
+description: This wizard will create a new CLI config profile for you
+plan:
+  decide_profile_type:
+    values:
+      config_type:
+        type: prompt
+        description: "What would you like to configure"
+        choices:
+          # There's also Web Identity and SAML but we're skipping that.
+          - display: Static Credentials
+            actual_value: static_creds
+          - display: Assume Role
+            actual_value: assume_role
+          - display: Process Provider
+            actual_value: proc_provider
+          - display: Additional CLI configuration
+            actual_value: additional_config
+      profile_name:
+        type: prompt
+        description: Enter the name of the profile
+    next_step:
+      switch: config_type
+      static_creds: get_static_creds
+      assume_role: get_assume_role
+      proc_provider: get_process_provider
+      additional_config: get_extra_cli_config
+  get_static_creds:
+    values:
+      access_key_id:
+        type: prompt
+        description: Enter your Access Key Id
+      secret_key:
+        type: prompt
+        description: Enter your Secret Access Key
+    next_step: DONE
+  get_process_provider:
+    values:
+      creds_process:
+        type: prompt
+        description: Enter the credential process command
+    next_step: DONE
+  get_assume_role:
+    values:
+      existing_profiles:
+        type: sharedconfig
+        operation: ListProfiles
+      existing_roles:
+        type: apicall
+        operation: iam.ListRoles
+        params: {}
+        query: "sort_by(Roles[].{display: RoleName, actual_value: Arn}, &display)"
+      role_arn:
+        type: prompt
+        description: Select the role you want to assume
+        choices: existing_roles
+      credential_source_type:
+        type: prompt
+        description: Select your credentials source type
+        choices:
+          - display: Source Profile
+            actual_value: source_profile
+          - display: Environment Variables
+            actual_value: Environment
+          - display: Amazon EC2 Instance Metadata
+            actual_value: Ec2InstanceMetadata
+          - display: Amazon ECS Container Credentials
+            actual_value: EcsContainer
+    next_step:
+      switch: credential_source_type
+      source_profile: get_source_profile
+      Environment: DONE
+      Ec2InstanceMetadata: DONE
+      EcsContainer: DONE
+  get_source_profile:
+    values:
+      source_profile:
+        type: prompt
+        description: Select the source profile
+        choices: existing_profiles
+    next_step: DONE
+  get_extra_cli_config:
```

Comment:
Maybe we should put a message to give people a heads up that it does not work right now?